src/pretalx/person/tasks.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.core.files import File
+from django.core.files.temp import NamedTemporaryFile
+from django.dispatch import receiver
+from requests import get
+
+from pretalx.celery_app import app
+from pretalx.common.signals import periodic_task
+from pretalx.person.models.user import User
+
+logger = logging.getLogger(__name__)
+
+
+@app.task()
+def gravatar_cache(person_id: int):
+    user = User.objects.filter(pk=person_id).first()
```

Comment:
`first()` is good, but then please add an early out in case this didn't return a user.